app/src/main/kotlin/com/wire/android/ui/common/bottomsheet/conversation/ConversationSheetContent.kt

```diff
@@ -102,16 +103,21 @@ data class ConversationSheetContent(
     val conversationId: ConversationId,
     val mutingConversationState: MutedConversationStatus,
     val conversationTypeDetail: ConversationTypeDetail,
-    val isSelfUserMember: Boolean = true,
+    val selfRole: Conversation.Member.Role?,
     val isTeamConversation: Boolean
 ) {
+
+    private val isSelfUserMember: Boolean get() = selfRole != null
+
     fun canEditNotifications(): Boolean = isSelfUserMember
             && ((conversationTypeDetail is ConversationTypeDetail.Private
             && (conversationTypeDetail.blockingState != BlockingState.BLOCKED))
             || conversationTypeDetail is ConversationTypeDetail.Group)
 
     fun canDeleteGroup(): Boolean =
-        conversationTypeDetail is ConversationTypeDetail.Group && isSelfUserMember && conversationTypeDetail.isCreator && isTeamConversation
+        conversationTypeDetail is ConversationTypeDetail.Group &&
+                selfRole == Conversation.Member.Role.Admin &&
+                conversationTypeDetail.isCreator && isTeamConversation
```

Comment:
so user can delete the group only if he's creator AND admin of that group? 🤔 
maybe we need to remove `&& conversationTypeDetail.isCreator` from here?

Reply:
yes, delete is only for team conversations and self is the creator and an admin

removing isCreator will display the delete option for all admins

which they clicked it will fail